DEPLOY_MODEL.md

```diff
@@ -10,11 +10,10 @@
 
 3. On your cluster dashboard, select `Worker Nodes` in the sidebar and take note of the `Public IP Address` for the node.
 
-> Note: The IP address will not be displayed until the cluster is fully deployed.
-
+   > Note: The IP address will not be displayed until the cluster is fully deployed.
```

Comment:
need the newline still otherwise the image is included in the "note"
```suggestion
   > Note: The IP address will not be displayed until the cluster is fully deployed.
   
```